tval

```diff
@@ -48,6 +48,10 @@ var metadata = {
 
 // The module will evaluate and have process.argv ['node', filename, ...]
 System.import(moduleName, {metadata: metadata}).catch(function(err) {
-  console.error('tval fails:',  err.stack || err + '');
+  if (err instanceof Array) {
+    console.log(err.join('\n'));
```

Comment:
Maybe not, I was behind.  I'll revert that.

I'm trying to see if I can compile this no-modules output and use it as traceur. Bootstrap hell, let me see if I can get out.